### ISSUE TITLE
Multiple output against a video

### ISSUE THREAD

**Issue by talhaanwarch** [facebookresearch/pytorchvideo#205]
In some cases we need multiple labels against each video. I can create a PR for this

**Comment by fcakyon** [comment]
@talhaanwarch would be great if you could update the dataloader for multilabel data!

**Comment by talhaanwarch** [comment]
Actually, i added the features #204 , but that is based on pandas. My use case is a column with video path and other columns with labels
